src/lib/components/auth/FeatureSlider.svelte

```
@@ -0,0 +1,185 @@
+    <button 
+      class="rounded-full bg-white/80 hover:bg-white p-2 transition-colors"
+      on:click={prevSlide}
+      disabled={transitioning}
+    >
+      <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+        <path d="m15 18-6-6 6-6"/>
+      </svg>
+    </button>
+
+    <div class="flex space-x-2">
+      {#each slides as _, index}
```

ESLint

[error] '_' is defined but never used.

(@typescript-eslint/no-unused-vars)